airbyte-integrations/connectors/source-gong/source_gong/manifest.yaml

```diff
@@ -211,6 +271,7 @@ metadata:
   autoImportSchema:
     users: false
     calls: false
+    extensiveCalls: false
```

Comment:
Did you add this intentionally, or because other streams also have it? I wonder if we really need to disable auto-import here.

Reply:
Ah yes I only did it because the others were like this. I think it makes sense tho cos this is a more detailed version of calls so if that's disabled it feels like this should be? Very happy to flip it tho of course